### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by guiworldtv2** [bibanon/tubeup#228] [Q&A] [answered]
whats up
i appreciated this idea
i run in a ubuntu the code

i stopped in:
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package yt-dlc

my account on internet archive come from my google account
how i can login on my account using codes?
i'm little confusing about how make this app works
thanks

**Comment by vxbinaca** [comment, marked as answer]
Please start at step one again this should be fixed now.

**Reply by guiworldtv2** [reply]
what is the command to put correctly the email and the password?

**Reply by vxbinaca** [reply]
Step 4.
